CONTRIBUTING.md

```diff
@@ -24,7 +24,7 @@ Before running the code, ensure you have the correct version of .NET installed (
 
 The next step is to ensure you have a config file for DAB defined. If you want to use the sample one [you can use the generator tool](#an-alternative-way-to-generate-config-files), otherwise, ensure that the `dab-config.<engine>.json` file is in the `src/Service` directory (and if the database is CosmosDb_NoSql the GraphQL schema file too).
 
-Make sure the config has a valid connection string, or you use the [`@env` syntax to reference an environment variable](docs/configuration-file.md#accessing-environment-variables).
+Make sure the config has a valid connection string, or you use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/en-us/azure/data-api-builder/configuration-file#accessing-environment-variables).
```

Comment:
```suggestion
Make sure the config has a valid connection string or use the [`@env` syntax to reference an environment variable](https://learn.microsoft.com/azure/data-api-builder/configuration-file#accessing-environment-variables).
```

removing `en-us` will allow browser to determine locale for non-US based readers.

Reply:
Thanks, excuse my oversight. Changed the URL without the locale.